watcher.go

```diff
@@ -49,9 +49,9 @@ func (m *MSG) UnmarshalBinary(data []byte) error {
 // setters allows for inline WatcherOptions
 //
 // 		Example:
-// 				w, err := rediswatcher.NewWatcher("127.0.0.1:6379",WatcherOptions{})
+// 				w, err := rediswatcher.NewWatcher("127.0.0.1:6379",WatcherOptions{}, nil)
 //
-func NewWatcher(addr string, option WatcherOptions) (persist.Watcher, error) {
+func NewWatcher(addr string, option WatcherOptions, optionalUpdateCallback func(string)) (persist.Watcher, error) {
```

Comment:
The method definition changed, could you put `optionalUpdateCallback` into `option` to fix the test error?

Reply:
Ok - those changes have been made and ready for re-review.